GeorgianMemenator/Services/BusinesssModels/CatTagDetail.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  CatTagDetail.swift
+//  GeorgianMemenator
+//
+//  Created by Giorgi Makadze on 31.01.2023.
+//  Copyright © 2023 Shalva Avanashvili. All rights reserved.
+//
+
+import Foundation
+
+struct CatTagDetail {
+    var uniqueID: Int
```

Comment:
Instead of having `uniqueId`, its better to conform this struct to Identifiable, like we are doing it in `MemeCategory`.

So,

```swift
struct CatTagDetail: Identifiable {
    let id: String = UUID().uuidString
    let name: String
}
```

After that you can have a list like that 👇 

```swift
List(viewModel.tags) { tag in
    Text(tag.name)
}
```